rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -79,28 +79,28 @@ NodeGraph::get_topic_names_and_types() const
 std::vector<std::string>
 NodeGraph::get_node_names() const
 {
-  rcl_string_array_t node_names_c =
-    rcl_get_zero_initialized_string_array();
+  utilities_string_array_t node_names_c =
+    utilities_get_zero_initialized_string_array();
 
   auto ret = rcl_get_node_names(node_base_->get_rcl_node_handle(),
       &node_names_c);
-  if (ret != RMW_RET_OK) {
-    if (rmw_destroy_node_names(&node_names_c) != RMW_RET_OK) {
+  if (ret != RCL_RET_OK) {
+    if (utilities_string_array_fini(&node_names_c) != RCL_RET_OK) {
```

Comment:
Shouldn't the return code be compared to one of these: https://github.com/ros2/c_utilities/blob/e6ff86450ddef392b18cb15a2bf5b165f77ca3d0/include/c_utilities/types/utilities_ret.h#L24-L26